data/174/607/204/7/1746072047.geojson

```diff
@@ -0,0 +1,59 @@
+{
+  "id": 1746072047,
+  "type": "Feature",
+  "properties": {
+    "edtf:cessation":"uuuu",
+    "edtf:inception":"uuuu",
+    "geom:area":0.003001,
+    "geom:bbox":"46.850172,24.502534,46.946893,24.585534",
+    "geom:latitude":24.543625,
+    "geom:longitude":46.900452,
+    "iso:country":"SA",
+    "lbl:latitude":24.544097,
+    "lbl:longitude":46.900452,
+    "mps:latitude":24.544097,
+    "mps:longitude":46.900452,
+    "mz:hierarchy_label":1,
+    "mz:is_current":1,
+    "mz:is_funky":0,
+    "name:ara_x_preferred":[
+        "\u0627\u0644\u0645\u062f\u064a\u0646\u0629 \u0627\u0644\u0635\u0646\u0627\u0639\u064a\u0629 \u0627\u0644\u062b\u0627\u0646\u064a\u0629"
+    ],
+    "name:eng_x_preferred":[
+        "2nd Industrial City"
+    ],
+    "name:eng_x_variant":[
```

Comment:
Ditto on the Arabic wrongly in the English variant name slot.

Reply:
fixed in 27ea53d